api/filtering/filtering_common.py

```diff
@@ -1,26 +1,38 @@
+from sqlalchemy import BigInteger
+from sqlalchemy import Boolean
+from sqlalchemy.sql.expression import ColumnOperators
+
 # Converts our filter param comparison operators into their SQL equivalents.
 POSTGRES_COMPARATOR_LOOKUP = {
-    "lt": "<",
-    "lte": "<=",
-    "gt": ">",
-    "gte": ">=",
-    "eq": "=",
-    "neq": "<>",
-    "is": "IS",
-    "contains": "?",
-    "nil": "IS NULL",
-    "not_nil": "IS NOT NULL",
+    "lt": ColumnOperators.__lt__,
+    "lte": ColumnOperators.__le__,
+    "gt": ColumnOperators.__gt__,
+    "gte": ColumnOperators.__ge__,
+    "eq": ColumnOperators.__eq__,
+    "neq": ColumnOperators.__ne__,
+    "is": ColumnOperators.is_,
+    "contains": "contains",
```

Comment:
Maybe use `ColumnOperators.contains`?
```suggestion
    "contains": ColumnOperators.contains,
```

Reply:
There actually isn't a ColumnOperator that checks for whether an array contains a value. `ColumnOperators.contains` checks for a substring within a string